### PR TITLE
Refinements

### PR DESCRIPTION
**Pull request by gillkyle**
Some of this is may be ugly but I'm just trying to crank through what I can, feel free to tear apart via code review

### CODE REVIEW

File: site/src/components/Button.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
 const Button = ({ children, href, radius = 0, toLink, node, ...props }) => {
   return (
     <button
       onClick={() => {
         // navigate to rel or external link if provided
-        node && window.location.assign(node.linkUrl)
-        toLink && navigate(toLink)
+        toLink && navigate(toLink);
+        node && window.location.assign(node.linkUrl);
```
Comment (jlengstorf) [inline site/src/components/Button.js:12]: This is less about correctness and more about legibility, but in general I'd prefer to see `if` statements in cases like these — it's a little trickier to understand what's going on here.

File: site/src/components/Figure.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Img from 'gatsby-image';
+import { getFluidGatsbyImage } from 'gatsby-source-sanity';
+
+// this wouldn't need to be shadowed if we add a prop to the Figure at the theme level
```
Comment (jlengstorf) [inline site/src/components/Figure.js:5]: Can you say more about what you ran into here?

File: site/src/components/Illustrations.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```
Comment (jlengstorf) [inline site/src/components/Illustrations.js:1]: `React` is only needed if you haven't set a pragma.
```suggestion
```

File: site/src/components/Illustrations.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        height="20"
+        width="20"
+      >
+        <ellipse cx="10" cy="10" rx="10" ry="10" fill="#FFE4A1" />
+      </svg>
+    </>
+  );
+};
+
+export default Illustrations;
```
Comment (jlengstorf) [inline site/src/components/Illustrations.js:46]: Kind of a general question here, but I wonder if this is better to add in CSS as background images. I don't think it matters enough to block us from moving forward, but in general it seems like design-only elements like these make more sense as background images.

File: site/src/gatsby-theme-marketing-sanity/components/serializers.js [hunk 5]
```diff
@@ -16,7 +17,17 @@ export default {
     button: Button,
     callout: Callout,
     code: CodeBlock,
-    'page-image': Figure,
+    'page-image': ({ node }) => (
```
Comment (jlengstorf) [inline site/src/gatsby-theme-marketing-sanity/components/serializers.js:20]: This feels pretty rough — we can merge this, but let's come back to it and make this better. Opened #13 to track this.

**Review by jlengstorf — APPROVED** [review]
Looks good! I'm going to merge, but I left a few comments.

File: site/src/components/Figure.js (hunk 2 above)
Comment (gillkyle) [reply site/src/components/Figure.js:5]: I ended up shadowing this so I could change the size and layout, the theme has a hardcoded maxWidth of ~540. If it accepted a prop for the maxWidth we could just import the component from the theme and pass in the maxWidth prop.

File: site/src/components/Illustrations.js (hunk 3 above)
Comment (gillkyle) [reply site/src/components/Illustrations.js:1]: TIL!

File: site/src/components/Illustrations.js (hunk 4 above)
Comment (gillkyle) [reply site/src/components/Illustrations.js:46]: That's true, getting these in right places was kind of tricky no matter how I sliced it but that could make a lot of sense.

**Pull request merged**